src/schema/datatypes/nirs.yaml

```diff
@@ -0,0 +1,49 @@
+---
+# fNIRS recordings
+- suffixes:
+    - nirs
+  extensions:
+    - .snirf
+    - .json
+  entities:
+    subject: required
+    session: optional
+    task: optional
+    run: optional
+# Events
+- suffixes:
+    - events
+  extensions:
+    - .tsv
+  entities:
+    subject: required
+    session: optional
+    task: optional
+    run: optional
+- suffixes:
+    - channels
+  extensions:
+    - .tsv
+  entities:
+    subject: required
+    session: optional
+    task: optional
+    run: optional
+- suffixes:
+    - coordsystem
+  extensions:
+    - .json
+  entities:
+    subject: required
+    session: optional
+    task: optional
+    run: optional
+- suffixes:
+    - optodes
+  extensions:
+    - .tsv
+  entities:
+    subject: required
+    session: optional
+    task: optional
```

Comment:
If we want to make this consistent with the *_electrodes.tsv, the *_optodes.tsv should not include the entity 'task' or 'run'. See also this discussion: https://github.com/bids-standard/bids-examples/issues/244

Reply:
Great point Helena. Thanks for highlighting that discussion (so many pieces of gold in the issues archive!). 

In summary we need to modify the optodes to mirror...

https://github.com/bids-standard/bids-specification/blob/78da1c8779a0bb30d042246cb15767d17526cbcd/src/schema/datatypes/eeg.yaml#L39-L48

Reply:
Updated. And I reorded the file to better match the eeg.yaml file. Thanks @helenacockx, good catch.